Gems/Atom/RHI/Code/Source/RHI/BufferView.cpp

```diff
@@ -29,9 +29,9 @@ namespace AZ
                 }
             }
 
-            m_hash = TypeHash64(buffer.GetHash(), m_hash);
             m_descriptor = viewDescriptor;
-            m_hash = m_descriptor.GetHash();
+            m_hash = TypeHash64(buffer.GetHash(), m_hash);
```

Comment:
Should this just be `m_hash = buffer.GetHash()`? Or did you mean to initialize `m_hash` to this object's address hash? (same with the change to the other file)

Reply:
yeah this can be m_hash = buffer.GetHash(). I will update it.